core/master/src/main/java/org/drftpd/commands/pre/PermissionPreHook.java

```diff
@@ -32,23 +32,23 @@
  */
 public class PermissionPreHook {
```

Comment:
Please use the java default code style of Idea and reformat your code. Dont want to have this kind of update when you push some code :)

Reply:
I think I fixed them all, please correct me if I'm wrong